app/views/courses/StudentRankingView.coffee

```diff
@@ -0,0 +1,198 @@
+helper = require 'lib/coursesHelper'
+require('app/styles/courses/student-ranking-view.sass')
+RootView = require 'views/core/RootView'
+template = require 'templates/courses/student-ranking-view'
+LevelSession = require 'models/LevelSession'
+CocoCollection = require 'collections/CocoCollection'
+Campaign = require 'models/Campaign'
+Level = require 'models/Level'
+utils = require 'core/utils'
+require 'three'
+UserPollsRecord = require 'models/UserPollsRecord'
+Poll = require 'models/Poll'
+CourseInstance = require 'models/CourseInstance'
+api = require 'core/api'
```

Comment:
required?

Reply:
It is dispensable. It will be removed on the next commit.